src/main/java/frc/robot/notification/LEDState.java

```diff
@@ -0,0 +1,64 @@
+package frc.robot.notification;
+
+public enum LEDState {
+  UNDERTAKER_DISABLED(0.61, "Undertaker Disabled", "#ff0000", -1), //RED
+  SPINNING_UP(0.69, "Motors spinning up", "#ffff00", -1), //YELLOW
```

Comment:
Some color libraries are picky with how hex colors are added I'm not sure if WPILib will complain because some of the values are capitalized and include hashes in the string.

Reply:
It worked with the wpilib sim stuff we are using it for, I needed to add the "#" before as well.